vault/resource_database_secret_backend_connection.go

```diff
@@ -907,19 +936,14 @@ func setInfluxDBDatabaseConnectionData(d *schema.ResourceData, prefix string, da
 	}
 }
 
-func setSnowflakeDatabaseConnectionData(d *schema.ResourceData, prefix string, data map[string]interface{}) {
+func setDatabaseConnectionData_userPass(d *schema.ResourceData, prefix string, data map[string]interface{}) {
```

Comment:
Underscores in function names are not standard in Go

```suggestion
func setDatabaseConnectionDataWithUserPass(d *schema.ResourceData, prefix string, data map[string]interface{}) {
```

Reply:
Fixed in 8f3da39